pom.xml

```diff
@@ -41,7 +41,7 @@
 
    <properties>
       <jacoco.outputDir>${project.basedir}/../../target/</jacoco.outputDir>
-      <hawkbit.version>0.3.0M2</hawkbit.version>
+      <hawkbit.version>0.3.0-SNAPSHOT</hawkbit.version>
```

Comment:
Just a reminder: Wait for `0.3.0M3` containing all the changes you need. 
Otherwise everybody out there have to build hawkBit first before you using any extension